src/video2x-qt6_pt_PT.ts

```diff
@@ -0,0 +1,518 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="pt_PT">
+<context>
+    <name>MainWindow</name>
+    <message>
+        <location filename="mainwindow.ui" line="14"/>
+        <source>Video2X</source>
+        <translation></translation>
+    </message>
+    <message>
+        <location filename="mainwindow.ui" line="29"/>
+        <source>Video Processing Job Queue</source>
+        <translation>Fila de Processos de Vídeo</translation>
+    </message>
+    <message>
+        <location filename="mainwindow.ui" line="43"/>
+        <source>Click the &quot;Add File(s)&quot; button or drag files into the job queue to add videos for processing.</source>
+        <translation>Clique no botão &quot;Adicionar Ficheiro(s)&quot; ou arraste os ficheiros para a fila de processos de vídeo.</translation>
```

Comment:
"Ficheiro" is more likely PT (from Portugal) than PTBR. Is that the objective?

Reply:
Yes, it is. I just saw the language preferences on the bottom. Ignore this comment.

Reply:
“Ficheiro” is more for Portuguese in Portugal, in Brazil they use the word ‘arquivo’, but Brazilians can understand it anyway.

> "Ficheiro" is more likely PT (from Portugal) than PTBR. Is that the objective?